string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,22 @@
+/* exported capitalizeWord */
+/*
+- create storage for output
+- make all characters upper case
+- go through each character one by one
+- starting at the second character, make all characters lower case
+- add each character to storage
+- return output
+ */
+
+function capitalizeWord(word) {
+  var output = '';
+  output = word[0].toUpperCase(word[0]);
+  for (var i = 1; i < word.length; i++) {
+    if (word.toUpperCase(word) !== 'JAVASCRIPT') {
+      output += word[i].toLowerCase(word[i]);
```

Comment:
I would do this conditional before the loop even begins, so you can simplify the work within this loop, and also avoid any extra actions being taken.